linux/test/app.test.js

```diff
@@ -27,5 +30,14 @@ describe('application', function () {
         assert.equal(count, 2);
       });
     }
+    // Prints all the logs produced by DEBUG
```

Comment:
Travis is failing because you are using a promise here (asynchronous call).
You'll need to pass in a `done` variable through the `it` block
```js
it('starts the application', (done) => {
  this.app.client.getMainProcessLogs().then((logs) => {
  // ..after code has executed
  done() // <-- let the test know you are done
  })
})

Reply:
Travis is failing on the OSX build, not the Linux one. I think it's just a timeout problem from what the logs say. I'll restart the build and see what happens. You can see that on [my fork](https://travis-ci.org/fcastilloec/headset-electron/builds/318527431), Travis built without any problems 
If it keeps failing on the OSX, could you test the OSX side on your machine? I'll try to do it on my VM. It might just be a problem with Travis itself.
Also, the test is passing, the error is in the `after` function

Reply:
Sure I'll test it. give me few minutes